service/population.go

```diff
@@ -55,3 +56,23 @@ func (p *populationService) GetPopulationStatistics() ([]model.PopulationRespons
 	}
 	return out, nil
 }
+
+func (p *populationService) GetAllCandidateInfo() ([]model.PopulationDatabaseRow, error) {
+	out := []model.PopulationDatabaseRow{}
+	districts, err := p.repository.QueryAllDistrict()
+	if err != nil {
+		logrus.Error(err)
+	}
```

Comment:
Why do you not return an error? Is there any reason?

Reply:
I think we don't have to return it. If there is an error it should be fine, it will returns the empty candidate list and log the err. Or it is better to return the error please lmk.

Reply:
fixed